test/test_formats.py

```diff
@@ -491,6 +489,11 @@ class MP3Test(ID3Test):
         #'podcast': '1',
         #'podcasturl': 'Foo',
         #'show': 'Foo',
+        'work': 'Foo',
+        'movementname': 'Bar',
+        'movementnumber': '1',
+        'movementtotal': '2',
+        'itunesgrouping': 'Foo',
```

Comment:
Please add tests to add and delete all these movement tags. Just adding them to the dict does not check if it gets deleted or not.